source/extensions/all_extensions.bzl

```diff
@@ -36,6 +36,9 @@ _core_extensions = [
     "envoy.network.dns_resolver.cares",
     "envoy.network.dns_resolver.apple",
     "envoy.load_balancing_policies.round_robin",
+    "envoy.http.original_ip_detection.xff",
+    "envoy.request_id.uuid",
+    "envoy.transport_sockets.tls",
```

Comment:
Given that all the real code is in `source/common/tls`, I'm not sure this is needed

Reply:
yeah I suspect with the reworking I can revert this.  I think it's a bug that we build code extensiosn but not required but arguably should go in a different PR